core/templates/dev/head/domain/learner_dashboard/learner_dashboard_icons_directive.html

```diff
@@ -9,6 +9,8 @@
    ng-if="belongsToLearnerPlaylist()"
    ng-click="removeFromLearnerPlaylist(getActivityId(), getActivityTitle(), getActivityType())"
    aria-hidden="true"
+   tooltip-enable="<[ playlistTooltipIsEnabled ]>"
```

Comment:
Remove spaces before `]>` and after `<[`.

Reply:
Done :+1: